manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'crm.settings')
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:

```

Comment:
Please use a more meaningful name here. There is nothing wrong with `exception `as a variable name.